### ISSUE TITLE
XCFramework crash - Symbol not found: _$s9Alamofire10HTTPMethodO3getyA2CmFWC - Xcode 14.3.1

### ISSUE THREAD

**Issue by NishanthMurugan** [Alamofire/Alamofire#3745]
We have pointed to old version of Alamofire 4.9.1 in podspec but it crashing on app launch 

  s.dependency 'Alamofire', '~> 4.9.1'

> Message from debugger: Terminated due to signal 6
dyld[51373]: Symbol not found: _$s9Alamofire10HTTPMethodO3getyA2CmFWC
  Referenced from: <F8A77ECB-739E-30C7-98E9-C95A6EF8FC77> /Users/king/Library/Developer/CoreSimulator/Devices/B0000F29-708F-463C-948B-526B0D7DE157/data/Containers/Bundle/Application/967FC5A3-2C71-4F3B-9AE3-F4CA019E1822/TestVSLib.app/Frameworks/WSLib.framework/VSLib
  Expected in:     <EE618A17-4E82-3F5E-9FCC-698F76A7E75E> /Users/king/Library/Developer/CoreSimulator/Devices/B0000F29-708F-463C-948B-526B0D7DE157/data/Containers/Bundle/Application/967FC5A3-2C71-4F3B-9AE3-F4CA019E1822/TestVSLib.app/Frameworks/Alamofire.framework/Alamofire

**Comment by jshier** [comment]
Sorry, this seems like some sort of build issue, not something we can help with.

**Comment by gj-thaoht** [comment]
Hi @NishanthMurugan, I have the same error. Please share with me how you solved it. Thank you.

**Comment by NishanthMurugan** [comment]
Hi @gj-thaoht  i faced this use when i use xcframework inside my private pod repository. This pod repository will consume in main project.

if we remove alamofire framework and comment the related codes, it works but we have less time to code do change, things i tried below

hardcoded the Alamofire version in xcframework and in my private pod and main project where we consume the private pod..  It failed same set of issue. Changed minimum deployment target also not useful .. 

I conveted xcframework to normal pod repository and generated the build. Not a better approach but hard to identify the exact fix
